ee/ucp/admin/configure/deploy-route-reflectors.md

```diff
@@ -274,3 +267,5 @@ IPv4 BGP status
 IPv6 BGP status
 No IPv6 peers found.
 ```
```

Comment:
Edit "You should see something like this" to read more like technical writing. :-)